src/utils/scripts/package.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Let's delete this `package.json` and instead add a new npm command in the original one. See examples for the `seed.ts` and `seedFirestore.ts` from the root `package.json`